### DISCUSSION TITLE
Data Analyzer

### DISCUSSION THREAD

**Discussion by 10xape** [TabularEditor/TabularEditor3#1409] [Data Modeling]
Hi

Where can I find information about the meaning of the data analyzer columns and how to implement any possible improvements from it ? Thank you 
![image](https://github.com/user-attachments/assets/5408bff4-d664-4f8b-945f-59b8558a694f)

**Comment by mlonsk** [comment]
Hi @10xape 

The Vertipaq Analyzer is a third-party tool created by SQLBI, and the best documentation that I am aware of exists on their home page: https://www.sqlbi.com/articles/data-model-size-with-vertipaq-analyzer/#:~:text=the%20Excel%20ribbon.-,Available%20Entities%20and%20Measures,-The%20data%20model

**Reply by 10xape** [reply]
thank you!
